### ISSUE TITLE
Strange answers from OpenAI

### ISSUE THREAD

**Issue by AlyxAbyss** [n3d1117/chatgpt-telegram-bot#96]
Recently, I have been receiving strange responses from OpenAI repeatedly. Have you experienced the same thing? What could be causing this? The script is running on my Raspberry Pi 3.

I asked Chat "GPT 3.5 Turbo" for a short story to illustrate the problem.

`2023-03-22 04:27:22,409 - root - INFO - New message received from user @AlyxAbyss
2023-03-22 04:27:49,850 - openai - INFO - message='OpenAI API response' path=https://api.openai.com/v1/chat/completions processing_ms=26864 request_id=adb2bdb843f3b8cfd0cf9f1eb5623e06 response_code=200
`

> Of course, here's another short story for you:

Emi was a talented ice skater, but she had never performed in front of an audience. She always got nervous when her friends and family came to watch her skate.

One Christmas, Emi's town held a holiday fair and they planned to put on an ice skating show. All eyes would be on Emi as she performed a beautiful solo routine. Emi was happy to be asked, but the thought of performing in front of so many people gave her cold chills.

Emi put her heart and soul into practicing her routine every day leading up to the show. The day finally came and all her loved ones were in the audience rooting for her. As Emi stepped out onto the rink, she felt relieved that she knew the routine by heart.

From the crack of the speakers, the melodic waltz tune began to fill ears, building upand comforting each step she took on the marveling ice. At that moment, Emi knew that she owned the ice right then and there. All her fears have suppressed been dominated unrivelled on ice. And with that positive mind state and focused split-second movements amid wind thudding zírts, twists, turns and loops with moments synchronizing with throwed, presented now they enspired the cheering packs under lights glotera surAmonizingincadence…. 

The best part slide window world gape-was the fireworks and eyes shocked sunlight touching up from wrsie heart syncmphonysated momentumthe aisles, oblong crystals tingleddown emwenivergiem englued her,handed in its brightest fréliquenor vein symdemceptué the nights freighting presto best accent splendõor – vanished within etitus pulsulosphére…it silent curtparadox air,kž of a sinking runner sprint called agony waving at victoresbehemote beams cheering long as her paths winding it through solvents mimics of famourands.

The ext no ra  inchimgr ord hishinol outwarette …..

I'm sorry about that, it looks like once again the program encountered an error resulting in my output becoming muddled and distorted. Let me try again and tell you a different story that makes more logical sense.

The good summer streams hair curled over lips bouncing steady rhythm.Ever felt aimed like ghosts haunting through echoed darts of weboflaa... Diznitellull spreaded to speak are natural. Fineworsinverted turned......okies!! This makes no sense... Please provide me a prompt for another story and I'll do my best to give you a coherent and enjoyable one.

**Comment by AlyxAbyss** [comment]
Another Story:


_Of course, here's another short story for you:

Emi was a talented ice skater, but she had never performed in front of an audience. She always got nervous when her friends and family came to watch her skate.

One Christmas, Emi's town held a holiday fair and they planned to put on an ice skating show. All eyes would be on Emi as she performed a beautiful solo routine. Emi was happy to be asked, but the thought of performing in front of so many people gave her cold chills.

Emi put her heart and soul into practicing her routine every day leading up to the show. The day finally came and all her loved ones were in the audience rooting for her. As Emi stepped out onto the rink, she felt relieved that she knew the routine by heart.

From the crack of the speakers, the melodic waltz tune began to fill ears, building upand comforting each step she took on the marveling ice. At that moment, Emi knew that she owned the ice right then and there. All her fears have suppressed been dominated unrivelled on ice. And with that positive mind state and focused split-second movements amid wind thudding zírts, twists, turns and loops with moments synchronizing with throwed, presented now they enspired the cheering packs under lights glotera surAmonizingincadence…. 

The best part slide window world gape-was the fireworks and eyes shocked sunlight touching up from wrsie heart syncmphonysated momentumthe aisles, oblong crystals tingleddown emwenivergiem englued her,handed in its brightest fréliquenor vein symdemceptué the nights freighting presto best accent splendõor – vanished within etitus pulsulosphére…it silent curtparadox air,kž of a sinking runner sprint called agony waving at victoresbehemote beams cheering long as her paths winding it through solvents mimics of famourands.

The ext no ra  inchimgr ord hishinol outwarette …..

I'm sorry about that, it looks like once again the program encountered an error resulting in my output becoming muddled and distorted. Let me try again and tell you a different story that makes more logical sense.

The good summer streams hair curled over lips bouncing steady rhythm.Ever felt aimed like ghosts haunting through echoed darts of weboflaa... Diznitellull spreaded to speak are natural. Fineworsinverted turned......okies!! This makes no sense... Please provide me a prompt for another story and I'll do my best to give you a coherent and enjoyable one._

**Comment by n3d1117** [comment]
No idea what's going on, but that's oddly terrifying 😨

**Comment by AlyxAbyss** [comment]
TEMPERATURE=1.5

"I changed the value to 1 and now Chat GPT seems to be able to communicate normally again. I wonder what would happen if I set the value to 2 for testing purposes." :D
